ci/cron/src/Main.hs

```diff
@@ -128,7 +127,7 @@ build_docs_folder path versions = do
     -- maps, and here order matters.
     let versions_json = versions
                         & map (\s -> "\"" <> s <> "\": \"" <> s <> "\"")
-                        & List.join ", "
+                        & List.intercalate ", "
```

Comment:
I removed the dependency on `MissingH` since we already depend on enough other kitchen sink libraries and it doesn’t buy us anything.